plugin/src/functionalTest/kotlin/com/exactpro/th2/gradle/Assertions.kt

```diff
@@ -0,0 +1,8 @@
+package com.exactpro.th2.gradle
```

Comment:
Could you please add copyright here and other files

Reply:
Yes, I will